core/ui/public/i18n/en/translation.json

```
@@ -1676,5 +1680,38 @@
         "available_with_subscription_title": "Activate subscription to enable this feature",
         "available_with_subscription": "Send a stream of security logs from applications and nodes of this cluster to Cloud Log Manager, where you can store, archive, and search them in a central place.",
         "cluster_id_notification": "Security logs from this cluster are available in the Cloud Log Manager UI under the host '{clusterId}'."
+    },
+    "applications": {
+        "title": "Applications",
+        "go_to_software_center": "Go to Software center",
+        "no_application": "No application installed",
+        "no_application_description": "You can find and install apps in the Software center.",
```

My bad, my previous suggestion was a little off, we now prefer the term "application" over "app"

```suggestion
        "no_application_description": "You can find and install applications in the Software center.",
```